src/test/scala/mesosphere/marathon/health/HealthCheckTest.scala

```diff
@@ -31,12 +30,35 @@ class HealthCheckTest extends MarathonSpec {
     assert(10 == proto.getGracePeriodSeconds)
     assert(60 == proto.getIntervalSeconds)
     assert(0 == proto.getMaxConsecutiveFailures)
+    assert(!proto.hasPort)
+  }
+
+  test("ToProtoOverridePort") {
+    val healthCheck = HealthCheck(
+      path = Some("/health"),
+      protocol = Protocol.HTTP,
+      portIndex = Some(0),
```

Comment:
Yes, we do that. This test only covers the serialization. Should we separate this to two different tests?